scripts/firedrake-install

```diff
@@ -688,6 +695,31 @@ def build_and_install_adjoint():
     install("dolfin-adjoint/")
 
 
+def build_and_install_slope():
+    log.info("Installing SLOPE\n")
+    if os.path.exists("SLOPE"):
+        slope_changed = git_update("SLOPE")
+    else:
+        git_clone("git+https://github.com/coneoproject/SLOPE.git")
+        slope_changed = True
+
+    if slope_changed:
+        os.chdir("SLOPE")
+        # Clean source directory
+        check_call(["git", "reset", "--hard"])
+        check_call(["git", "clean", "-f", "-x", "-d"])
+        # Build and install (hack: need to do this manually as SLOPE is
+        # still stupid and doesn't have a proper configure+make+install system)
+        check_call(["make"])
+        check_call(["mv"] + glob(os.path.join('lib', '*')) + [os.path.join(sys.prefix, 'lib')])
+        include_dir = os.path.join(sys.prefix, 'include', 'SLOPE')
+        check_call(["mkdir", "-p", include_dir])
+        check_call(["cp"] + glob(os.path.join('sparsetiling', 'include', '*')) + [include_dir])
+        # Also install the Python interface
+        run_cmd(pyinstall)
```

Comment:
Can you use the "install" command instead?  That way, pyslope will be installed using pip and will pick up "developer" flags and similar.